tests/Aquifer.Common.IntegrationTests/Utilities/VersificationUtilitiesTests.cs

```diff
@@ -0,0 +1,133 @@
+﻿using Aquifer.Common.Services.Caching;
+using Aquifer.Common.Utilities;
+
+namespace Aquifer.Common.IntegrationTests.Utilities;
+
+public sealed class VersificationUtilitiesTests(App _app) : TestBase<App>
+{
+    private readonly ICachingVersificationService _versificationService = _app.GetRequiredService<ICachingVersificationService>();
+
+    private const int _engVersificationSchemeBibleId = 0;
+    private const int _bsbBibleId = 1;
+    private const int _ls1910BibleId = 6;
+    private const int _rsbBibleId = 9;
+    private const int _gltBibleId = 10;
+
+    [Theory]
```

Comment:
For clarity, these integration tests are all hitting the Dev/QA DB, loading actual Bible versifications, exclusions, and max chapter/verse numbers, and using the actual `CachingVersificationService` code.